bundles/framework/featuredata/PopupHandler.js

```diff
@@ -0,0 +1,120 @@
+import { FEATUREDATA_BUNDLE_ID } from './view/FeatureDataContainer';
+import { showSelectToolPopup } from './view/SelectToolPopup';
+/**
+ * @class Oskari.mapframework.bundle.featuredata.PopupHandler
+ *
+ * Handles map selection popup functionality.
+ */
+Oskari.clazz.define('Oskari.mapframework.bundle.featuredata.PopupHandler',
```

Comment:
This doesn't need to be defined as Oskari.clazz. We should just create something that exports the functions that we can import where needed.